README.md

```diff
@@ -19,6 +19,20 @@ that incorporates and documents best practices for a performant and scalable des
 security by default for control, logging and evidence generation. It can be  simply deployed by
 customers through a Terraform workflow.
 
+## Security Recomendations
+
+We recommend you remove all primitive `owner` roles in the projects used as inputs for the *Data Warehouse module*.
+
+- This recommendation aligns with the principle of [separation of duties](https://cloud.google.com/kms/docs/separation-of-duties).
+
+You can check the current situation of your project:
+
+- Using [Security Health Analytics](https://cloud.google.com/security-command-center/docs/concepts-vulnerabilities-findings#security-health-analytics-detectors), checking the [KMS vulnerability findings](https://cloud.google.com/security-command-center/docs/concepts-vulnerabilities-findings#kms-findings), for the Detector `KMS_PROJECT_HAS_OWNER`. This is a *Premium* tier detector.
+  - You can search for the SHA findings with category `KMS_PROJECT_HAS_OWNER` in the Security Command Center in the  Google Cloud Console.
```

Comment:
if you are going to use an acronym (SHA) you need to indicate it right after the first reference on the previous line.